<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simulationStatus</class>
 <widget class="QDialog" name="simulationStatus">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>356</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Functional block simulation status</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icon_SystemLabDesigner.bmp</normaloff>Icon_SystemLabDesigner.bmp</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>502</width>
     <height>310</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>324</y>
     <width>115</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Simulation progress:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>326</y>
     <width>375</width>
     <height>19</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
